x-pack/plugin/sql/src/test/resources/org/elasticsearch/xpack/sql/planner/querytranslator_tests.txt

```diff
@@ -40,9 +40,26 @@ CONTAINS "term":{"int":{"value":5
 
 TermEqualityForDate
 SELECT some.string FROM test WHERE date = 5;
-"term":{"date":{"value":5
+"range":{"date":{"gte":"1970-01-01T00:00:00.005Z","lte":"1970-01-01T00:00:00.005Z"
 ;
 
+TermEqualityForDateWithConversion
```

Comment:
Would be nice to add some more variance in the tests (`now-1y` etc.)

Reply:
I couldn't find a way to test "now" expressions in this context, since the expected result would change at every execution and I think there is no way to make it dynamic.
Any suggestion is welcome of course.